src/WebSdk/Publish/Targets/PublishTargets/Microsoft.NET.Sdk.Publish.FileSystem.targets

```diff
@@ -66,11 +66,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <RemoveDir
       Directories="$(PublishUrl)"
-      Condition="'$(DeleteExistingFiles)' == 'true' And Exists('$(PublishUrl)')" />
+      Condition="'$(DeleteExistingFiles)' == 'true' And '$(PreserveDestinationFolder)' == 'false' And Exists('$(PublishUrl)')" />
```

Comment:
This may need to be '$(PreserveDestinationFolder)' != 'true'. According to this documentation, both 'false' and unset would be != to 'true'.

https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2019